Dashboard.tsx Cleanup.

changeling opened the pull request:
Corrected minor word usage.

Re-added public/assets directory and added Public Domain images for Dashboard mockup persons.

Updated PersonCard dates and image values to include birth/death dates and included images.

briot commented on src/Dashboard.tsx:
```diff
@@ -93,10 +93,10 @@ class Dashboard extends React.PureComponent<{}, {}> {
+               <PersonCard name="Emmanuel Briot" dates="1975-" image="/assets/Emmanuel_Briot.jpg"/>
+               <PersonCard name="John Smith" dates="~January, 1580-June 21, 1631" image="/assets/John_Smith.jpg" />
+               <PersonCard name="Louis XIII" dates="September 27, 1601-May 14, 1643" image="/assets/Louis_XIII.jpg" />
+               <PersonCard name="Josephine" dates="June 23, 1763-May 29, 1814" image="/assets/Josephine.jpg"/>
```
I had left the "Josephine" card with an invalid image and no dates on purpose. These are data that might not be available for some persons, and the cards should still work in that context. Can we still have some nice layout even if there's no image (which is likely to be the vast majority of persons in practice) ? For the same reason, I had a card with an image with no date, though I realize I was missing a card with no image but with dates.

If you are interested in design, would you like to experiment with this ?

briot replied:
Also, my image can't be mine, I don't have a twitter profile :-) You can use the one from github, or you can use any random image from the web, I don't mind !

changeling commented:
OK, I've committed fixes to address these. Take a look at the updated pull request and see if that looks better. I updated my card changes and added a few that seem to provide good coverage.

> If you are interested in design, would you like to experiment with this ?

I might play around somewith that.. What are your particular thoughts on that? Layout/CSS, possible issues, all of the above? :)

Oh, and I apologize got assuming the Twitter Emmanuel Briot was you. :/ AND for taking the liberty of adding a photo. Never assume and always ask. I need to remember that. :)

briot commented:
The final version looks good. The change for resp.status was already part of another pull request and has been merged already, so likely it isn't necessary. Would you merge some of the commits, so that we don't get the old ones before my review, then the new ones you just did, that change those ? In fact, since they all mostly impact the same files, perhaps it makes sense to merge some of them into a single slightly large commit..

Thanks !

briot commented:
> Trying to figure out to how to squash into a single commit. And not sure why the resp.status is being included here. Is this a situation where I should revert and recommit, or is there a method to retroactively squash the commits?

I think github takes the whole branch, not just the specific patch you specified for the pull request.

Use `git rebase -i HEAD~12` (or something else than 12) for interactive rebase. Change the first word of
each line with “f"
